UnitsNet/UnitDefinitions/HeatFlux.json

```diff
@@ -0,0 +1,117 @@
+﻿{
+  "Name": "HeatFlux",
+  "BaseUnit": "WattPerSquareMeter",
+  "XmlDoc": "Heat flux is the flow of energy per unit of area per unit of time",
+  "Units": [
+    {
+      "SingularName": "WattPerSquareMeter",
+      "PluralName": "WattsPerSquareMeter",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Prefixes": [ "Nano", "Micro", "Milli", "Centi", "Deci", "Kilo" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "W/m²" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "WattPerSquareInch",
+      "PluralName": "WattPerSquareInch",
+      "FromUnitToBaseFunc": "x*1.5500031e3",
+      "FromBaseToUnitFunc": "x/1.5500031e3",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "W/in²" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "WattPerSquareFoot",
+      "PluralName": "WattPerSquareFoot",
+      "FromUnitToBaseFunc": "x*1.07639e1",
+      "FromBaseToUnitFunc": "x/1.07639e1",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "W/ft²" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "BtuPerSquareInchSecond",
+      "PluralName": "BtuPerSquareInchSecond",
+      "FromUnitToBaseFunc": "x*1.63533984e6",
+      "FromBaseToUnitFunc": "x/1.63533984e6",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "BTU/in²/s" ]
```

Comment:
Would `BtuPerSecondSquareInch` and `BTU/(s∙in²)` be better? Ref comments above about naming.
Same for units below.

Reply:
Yes, I will change this.